Neos.Flow/Classes/ObjectManagement/Configuration/ConfigurationBuilder.php

```diff
@@ -353,6 +373,51 @@ protected function parseArgumentOfTypeObject($argumentName, $objectNameOrConfigu
         return $argument;
     }
 
+    /**
+     * Creates a "virtual object configuration" for object arguments, turning:
```

Comment:
Why do we need to convert all those old object configurations into virtual object configurations?

Reply:
We only convert those that didn't work before (i.e. when there are nested factories configured):
```yaml
Some\Class:
  factoryObjectName: Some\Factory
  arguments:
    1:
      object:
        factoryObjectName: Some\Other\Factory
```
This configuration would trigger a warning:
```
Warning: Illegal offset type in Packages/Framework/Neos.Flow/Classes/ObjectManagement/ObjectManager.php line 198
```
because in the `ObjectManager::buildObjectByFactory()` the `$argumentInformation['v']` for the first argument contains an instance of `ObjectConfiguration`.. so the part `$this->get($argumentInformation['v'])` fails..

TL;DR: For existing installations we actually don't replace anything here because it is only triggered for cases that didn't work before